### ISSUE TITLE
Functional contact page

### ISSUE THREAD

**Issue by AmaniLouendriz** [AmaniLouendriz/PontsDeConnaisssance#12]
Hello @caseyli ,

So I had a question on how to make the contact page functional, I made a kind of form-handler but for some reason it isn't working, I might be missing up things but is it like because I don't have a whole server on which the website stands that it can't send the email? And are there any good ways to do what I want to achieve? Thank you !

**Comment by caseyli** [comment]
@AmaniLouendriz you're totally right. Most of what you've been coding so far is static HTML and CSS that can only run in the browser. To make a contact form functional, you have to use some Server side language/framework. At this point, your learning of full stack frameworks really kicks in.

There are few options you can explore

- PHP
- Ruby on Rails
- Next.js
- Express
- Others

There's a lot to learn with each of these, but if you're still interested in Web Development, I would start with one of those.

Btw, I also offer some courses now: https://www.coursesxcasey.com/
